packages/http-client-csharp/eng/scripts/Add-Debug-Profile.ps1

```diff
@@ -156,6 +156,70 @@ function Get-ProfileName {
     return $dirName -replace '[^a-zA-Z0-9\-_.]', '-'
 }
 
+# Copy local generator DLLs to the target node_modules location
+function Copy-LocalGeneratorDlls {
+    param(
+        [string]$SdkPath,
+        [string]$PackageName
+    )
+    
+    # Get the script location and calculate the generator path
+    $scriptDir = Split-Path $MyInvocation.PSCommandPath -Parent
+    $packageRoot = Split-Path (Split-Path $scriptDir -Parent) -Parent
+    $generatorRoot = Join-Path $packageRoot "generator"
+    
+    # Project names and their corresponding DLL names
+    $projects = @(
+        @{ ProjectPath = "Microsoft.TypeSpec.Generator\src"; DllName = "Microsoft.TypeSpec.Generator.dll" }
+        @{ ProjectPath = "Microsoft.TypeSpec.Generator.ClientModel\src"; DllName = "Microsoft.TypeSpec.Generator.ClientModel.dll" }
+        @{ ProjectPath = "Microsoft.TypeSpec.Generator.Input\src"; DllName = "Microsoft.TypeSpec.Generator.Input.dll" }
+    )
+    
+    # Define target directory in node_modules
+    $targetDir = Join-Path $SdkPath "TempTypeSpecFiles\node_modules\@azure-typespec\$PackageName\dist\generator"
+    
+    # Create target directory if it doesn't exist
+    if (-not (Test-Path $targetDir)) {
+        New-Item -Path $targetDir -ItemType Directory -Force | Out-Null
+        Write-Host "Created directory: $targetDir" -ForegroundColor Yellow
+    }
+    
+    # Try to find and copy local DLLs
+    foreach ($project in $projects) {
+        $projectDir = Join-Path $generatorRoot $project.ProjectPath
+        $dllName = $project.DllName
+        
+        # Possible locations for the DLL (in order of preference)
+        $possiblePaths = @(
```

Comment:
Why wouldn't we just use the local dist as instructed? This should have all the dlls.

Reply:
No, you misunderstood. We still need to copy the 3 dlls I mentioned. But we should be using the local dist folder as the source for the copy.